src/solver/modeler/api/include/antares/solver/modeler/api/mipSolution.h

```diff
@@ -51,6 +53,7 @@ class IMipSolution
     virtual double getObjectiveValue() const = 0;
     virtual double getOptimalValue(const IMipVariable* var) const = 0;
     virtual std::vector<double> getOptimalValues(const std::vector<IMipVariable*>& vars) const = 0;
+    virtual const std::map<std::string, double>& solutionValues() const = 0;
```

Comment:
try to keep same naming
```suggestion
    virtual const std::map<std::string, double>& getOptimalValues() const = 0;
```